iosApp/iosApp/ContentView.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+import SwiftUI
+import shared
+
+
+struct ComposeView: UIViewControllerRepresentable {
+    func makeUIViewController(context: Context) -> UIViewController {
+        MainViewControllerKt.mainViewController()
+    }
+
+    func updateUIViewController(_ uiViewController: UIViewController, context: Context) {}
+}
+
+struct ContentView: View {
+    var body: some View {
+        ComposeView()
+                .ignoresSafeArea(.keyboard)
```

Comment:
What does this mean?

Reply:
I don't think is necessary. I have read more about it and I think, we can remove it. 
What it means is, when there is a trigger to a keyboard in a multi-platform project, Compose handles the keyboard positioning and resizing the content when the keyboard is shown or hidden. So there's no need to trigger any repositioning from IOS. 

I will go ahead and remove it and request a review again before meging